Shared/SharedUI/BaseDialogs/PackFileTree/MultiSelectTreeView.cs

```diff
@@ -0,0 +1,201 @@
+﻿using System;
```

Comment:
this file is never used?
What is multi select needed for? While its nice, it also adds a lot of new cases that needs to be handles all through the program, such as drag and drop handling

Reply:
I assume you want to select multiple audio files from a packfile for audio modding purposes? 
I suggest handling that special case directly in the audio editor. An example of a simple way to do this would be: 
1. Use the folder browser to let the user select folder
2. Based on folder, spawn a new dialog with a filelist where the user can select the actual files. 

WHile this is not as nice, its exponentially easier to manage from a development perspective, with a minimal inconvenience for users compared to the original idea.

Reply:
I added it in this PR, so it's separated from the Audio stuff for clarity. Would have then use in there once merged.

Yes selecting multiple wav files from the packfile is the goal, rather than the selecting it from the PC via the windows dialog. This is because packfile paths are relative and not local to the device so using audio projects anywhere but the PC they were made on would be a huge pain in the ass.

Those reservations seem sensible. The double dialog setup seems a little clunky though of course happy to do that if that's your preference. I assume piggybacking the original pack file browsing setup copying it into the AudioEditor and tweaking it for multi select is out of the question?